website/docs/schema-generator/writing-schemas/unions.md

```diff
@@ -43,3 +48,57 @@ type Query {
   whichHand(whichHand: String!): BodyPart!
 }
 ```
+
+## `@GraphQLUnion`
```

Comment:
can you add `known limitations` (or issues/or something like that) and list that it is not possible to declare union directives?